kasa/tests/conftest.py

```diff
@@ -53,6 +53,7 @@
     "KP115",
     "KP125",
     "KP401",
+    "KS200M",
```

Comment:
Should this also be a dimmer?

Reply:
KS200M has a PIR sensor, but doesn't have any dimming functionality.

https://www.kasasmart.com/us/products/smart-switches/kasa-smart-wifi-light-switch-motion-activated-ks200m

Reply:
Okay, thanks for the clarification, let's merge this too! :+1: